src/content/guides/creating-digital-tokens.mdx

```diff
@@ -132,4 +132,4 @@ sequenceDiagram
 
 4. **Transfer Token**
 
-    For sending tokens to users, see [Sending Assets](/guides/loading-and-sending-assets/#sending-assets)
```

Comment:
@Wulingoh You're breaking this link - you need to add redirects for any broken links as users might have been given or bookmarked the old ones

Reply:
@Wulingoh Have you tested this? I can't see the redirect working on the dev preview site

Reply:
@tom-blake, yes I have tested and ran the branch on dev-preview again. The redirect is doing what it is expected. http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/guides/transferring-assets/